src/RosNode.test.ts

```diff
@@ -10,15 +10,15 @@ describe("RosNode", () => {
   it("Publishes and subscribes to topics and parameters", async () => {
     const rosMaster = new RosMaster(new HttpServerNodejs());
     await rosMaster.start("localhost");
-    const rosMasterUri = rosMaster.url() as string;
+    const rosMasterUri = rosMaster.url()!;
     expect(typeof rosMasterUri).toBe("string");
 
     let errA: Error | undefined;
     let errB: Error | undefined;
 
     const nodeA = new RosNode({
       name: "/nodeA",
-      hostname: "localhost",
+      hostname: "0.0.0.0",
```

Comment:
was it a suggestion from lint?

Reply:
That was the test that kept failing and Alexandre was able to realize that changing the hostname to localhost was a fix to that error